analysis/report_templates/default.html

```diff
@@ -388,6 +388,18 @@ <h2 class="green-text text-darken-3">experiment data</h2>
             <br><br>
             The experiment was conducted using this FuzzBench commit:
             <a href="https://github.com/google/fuzzbench/commits/{{ experiment.git_hash }}">{{ experiment.git_hash }}</a>
+
+            <br><br>
+            To reproduce this experiment run the following commands in your FuzzBench repo:</a><br>
+            <code>
+            git checkout {{ experiment.git_hash }}  # Check out the right commit. <br>
```

Comment:
move "# Check out the right commit." to previous line to match others.

Reply:
Done.